cli/internal/runsummary/spaces.go

```diff
@@ -28,9 +28,7 @@ type spacesRunPayload struct {
 	Client         spacesClientSummary `json:"client"`                   // Details about the turbo client
 	GitBranch      string              `json:"gitBranch"`
 	GitSha         string              `json:"gitSha"`
-
-	// TODO: we need to add these in
-	// originationUser string
+	User           string              `json:"originationUser"`
```

Comment:
Same as above, do we want `"originationUser": ""` in the payload?

Reply:
Miiight be a good candidate for omitempty her actually. @gaspar09 what do you think? I think if we omit it, the backend will use the token to associate with a person, which seems good.